### PR TITLE
#169282179 Builds App component which fetches the API

### PR DESCRIPTION
**Pull request by b0nbon1**
#### What does this PR do?
 It creates app components which will contain the data fetched from the API. then filters the API data so as to pass necessary data
#### Description of Task to be completed?
- [x] add app component
- [x] fetch the API
- [x] filter data
- [x] pass necessary data
#### How should this be manually tested?
- Git clone the repo
- run npm install or yarn install
- Run npm start or yarn start
- go to your browser to view
#### Any background context you want to provide?
- No feature implemented yet
#### What are the relevant pivotal tracker stories?
- [#169282179](https://www.pivotaltracker.com/n/projects/2407431/stories/169282179)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
...
-#### How Has This Been Tested?
-#### Checklist:
-#### PIVOTAL TRACKER
+#### What does this PR do?
+#### Description of Task to be completed?
+#### How should this be manually tested?
+#### Any background context you want to provide?
+#### What are the relevant pivotal tracker stories?
+#### Screenshots (if appropriate)
+#### Questions:
```
Comment (kiproping) [inline .github/pull_request_template.md]: This should not be part of this PR. It should be a separate PR.

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "moment": "^2.24.0",
```
Comment (kiproping) [inline package.json]: Explain in your PR why you are introducing this package.

File: src/helpers/filterWeather.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const filterData = (items) => {
+    const daysEdit = [];
+    items.forEach((el) => {
+      const obj = { dt: convertTime(el.dt) };
+      Object.assign(el, obj);
+      if (!daysEdit.some((ele) => ele.dt === el.dt)) {
+        daysEdit.push(el);
+      }
+    });
+    return daysEdit;
```
Comment (kiproping) [inline src/helpers/filterWeather.js]: Thoroughly document this funcionality. What is going on here?

File: .github/pull_request_template.md (hunk 1 above)
Comment (kipropbrian) [inline .github/pull_request_template.md]: This should be a separate PR. It has no relation to your work

File: package.json (hunk 2 above)
Comment (kipropbrian) [inline package.json]: You should explain why you are introducing this package in your PR description

File: src/helpers/filterWeather.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+export {
+  convertTemperature,
+  filterData
+};
```
Comment (kipropbrian) [inline src/helpers/filterWeather.js]: Thoroughly document this function, so that we know what it is doing.

Comment (b0nbon1) [reply src/helpers/filterWeather.js]: Like writing docStrings or comments??

File: .github/pull_request_template.md (hunk 1 above)
Comment (b0nbon1) [reply .github/pull_request_template.md]: Absolutely, I'll change that

File: package.json (hunk 2 above)
Comment (b0nbon1) [reply package.json]: oh sorry, I actually removed because i needed it but Gildas told me we can use `new Date` to convert unix time to date time

File: src/helpers/filterWeather.js (hunk 4 above)
Comment (nshaphan) [reply src/helpers/filterWeather.js]: Some comments needed
